### ISSUE TITLE
Router not updating URL or setting state when VueMultianalytics is enabled

### ISSUE THREAD

**Issue by fnfilho** [Glovo/vue-multianalytics#3]
Hey guys!

We are getting ready to use VueMultianalytics in the next few days, but when setting it in our project we noticed vueRouter would start to not update the URL or set the browser state so the back button would not work anymore.

Heres our simplified main.js. Commenting the lines about VueMultianalytics will bring the functionality to work correctly. What am I doing wrong here?

```
import Vue from 'vue'
import VueMultianalytics from 'vue-multianalytics'
import VueRouter from 'vue-router'

import Component1 from 'components/component1'
import Component2 from 'components/component2'
import RootApp from './RootApp'

// Router configurations
// -------
const router = new VueRouter({
  mode: 'history',
  routes: [
    {
      path: '/',
      component: Component1
    },
    {
      path: '/company',
      component: Component2
    }
  ]
})

// Analytics
// -------
import {analytics} from './analytics'

Vue.use(VueMultianalytics, {
  modules: {
    ga: analytics.gaConfig,
    mixpanel: analytics.mixpanelConfig
  },
  routing: {
    vueRouter: router
  }
})

// Initialize
// -------
const app = new Vue({
  router,
  el: '#root',
  template: '<RootApp/>',
  components: { RootApp }
}).$mount()
```

**Comment by fnfilho** [comment]
Guy guys, any updates on this?

Thanks!

**Comment by Frzurita** [comment]
Hi fnfilho! 

First of all, thanks for report us this issue. My apologize for this late answer. 

The problem has been solved in the version 1.3.9 feel free to test it!

let me know if it's working for you just to doble check and then I'll close the issue :)

**Comment by fnfilho** [comment]
Just tested it! Working great! Thanks.
